caluma/caluma_workflow/jexl.py

```diff
@@ -123,10 +153,23 @@ def _get_dynamic_group(
 
 
 class FlowJexl(JEXL):
-    def __init__(self, **kwargs):
+    def __init__(
+        self, case=None, user=None, prev_work_item=None, dynamic_context=None, **kwargs
+    ):
         super().__init__(**kwargs)
-        self.add_transform("task", lambda spec: spec)
-        self.add_transform("tasks", lambda spec: spec)
+
+        self.add_transform(
+            "task",
+            lambda slug: self.task_transform(
+                slug, case, user, prev_work_item, dynamic_context
```

Comment:
Instead of (ab)using closures here, I'd prefer going the OO path and putting the `__init__()` parameters into instance properties (`self.case = case; self.user = user` etc). Then the `add_transform()` won't need a lambda expression anymore:
```python
self.add_transform("task", self.task_transform)
```
which in my opinion is a lot easier to debug and understand

Reply:
Good input, I changed it.

Reply:
Changed it for the groups transform as well: #1138